src/home/Footer.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./Footer.css";
+import { Link } from "react-router-dom";
+import github from "./../assets/images/github.svg";
+import linkdin from "./../assets/images/LinkedIn.svg";
+import twitter from "./../assets/images/Twitter.svg";
+import facebook from "./../assets/images/facebook.svg";
+export default function Footer() {
+  return (
+    <section id="footer">
+      <div className="footer-container">
+        <div className="main-flexbox">
+          <div className="flexbox-item">
+            <img className="logo" src="#" alt="Logo"></img>
```

Comment:
@naveen8801 `<img/>` tags are self closing. No need for closing tags.

Reply:
Sure :)